devel/sn-demangler/Portfile

```diff
@@ -29,7 +29,8 @@ java.fallback       openjdk11
 
 use_configure       no
 
-compiler.blacklist  *gcc*
+# Scala-Native requires clang 6+ or apple's clang 8+
+compiler.blacklist  *gcc* {clang < 800} {macports-clang-3.[0-9]} {macports-clang-[4-5].0}
```

Comment:
please see the `port lint` error message:
`Error: Line 33 uses compiler.blacklist entry {clang < 800} {macports-clang-3.[0-9]} {macports-clang-[4-5].0} which requires the compiler_blacklist_versions portgroup which has not been included`

Reply:
@reneeotten thanks, forgot to stage this!